### PR TITLE
Added Island Cake configs

### PR DESCRIPTION
**Pull request by Caedis**
Disabled by default
Target dim = void world (119)

### CODE REVIEW

File: overrides/config/DimensionalEdibles.cfg [hunk 1]
```diff
@@ -314,6 +317,42 @@ dimensionaledibles {
...
+
+        islandcake {
+            # Allow players to create personal islands when on a FTB Team
+            B:allowPersonalIslands=true
+
+            # Set to true to make the Island Cake consume fuel.
+            B:consumesFuel=true
+
+            # Set the fuel used by Island Cake (Don't change this unless you know what you're doing).
+            S:fuel=minecraft:cobblestone
```
Comment (IntegerLimit) [inline overrides/config/DimensionalEdibles.cfg:335]: Can this be changed to pulsating dust?

File: overrides/config/DimensionalEdibles.cfg [hunk 2]
```diff
@@ -20,6 +20,9 @@ dimensionaledibles {
 
         # Set to true to enable Ender Apple.
         B:enderApple=false
+        
+        # Set to true to enable Island Cakes.
+        B:islandCake=false
```
Comment (IntegerLimit) [inline overrides/config/DimensionalEdibles.cfg:25]: Should this be true?

**Review by IntegerLimit — CHANGES_REQUESTED** [review]
Can you add this information to the quest book?

File: overrides/config/DimensionalEdibles.cfg (hunk 1 above)
Comment (Caedis) [reply overrides/config/DimensionalEdibles.cfg:335]: Yep, just change it to the internal name

**Comment by Caedis** [comment]
The best way to handle is for the server admin to enable them, use the default void cake to go to 0,0 in the void dim, then place down some infinite use island cakes, including some personal island cakes.

**Comment by Caedis** [comment]
Personal island cakes are made by right clicking a normal island cake with the `personalLockingItem` item

**Comment by IntegerLimit** [comment]
It would be nice if you could change the fuel quickly, and add a README or qb so that the server admin can easily know about this. Otherwise, its great!

**Review by IntegerLimit — APPROVED** [review]

**Pull request merged**